### ISSUE TITLE
Color Contrast Fails for MongoDB Text (Grey on White Background) with 2.303:1 which is less than 4.5:1: A11y_ Visual Studio Code Services_Open Source AI code editor screen _Contrast Minimum.

### ISSUE THREAD

**Issue by AnKushSingh05** [Microsoft/vscode-docs#9313]
### GitHub Tags: 
#A11yTCS; #A11yMAS; #A11ySev2; #AI Limited; #DesktopWeb; #ChromiumEdge; #Win11; #Visual Studio Code Services; #E2E_Visual Studio Code Services_Web_Jan26; #WCAG1.4.3; #Contrast Minimum; #ES Chat;   
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification." 
Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Faka.ms%2Feschat-ask%3Fq%3DDoes%2BMAS%2B1.4.3%2Bapply%2Bto%2Belements%2Bin%2Bother%2Bstates%2B(such%2Bas%2Bfocus%2Bor%2Bhover)%3F&data=05%7C02%7Cv-yaja%40microsoft.com%7Cab1c8bf7fc40479470f408de5d149d45%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050540322712738%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=h0pesliWcxZ8OIISTmTyucruKF1rvvoJGsCM7BdUOa0%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fchannel%2f19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2fCAI%2520Accessibility%2520Support%2520Channel?groupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221198042%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2foutlook.office.com%2fbook%2fmicrosoft2779007%2540microsoft.onmicrosoft.com%2fs%2fbLeGBltMW0qI0WRLsTqx2A2?ismsaljsauthenabled&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221214666%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0). 
### Environment Details: 
Application Name: Visual Studio Code Services 
URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)  
Chromium Edge: Version 144.0.3719.104 (Official build) dev (64-bit) 
Edition: Windows 11 Enterprise Insider Preview 
Version: 24H2 
OS build: 26558.1000 
### Repro steps: 

1. Hit the URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0) 
2. Tab navigate to Code with Extensions section. 
3. Observe that Color Contrast Fails for MongoDB Text (Grey on White Background) with 2.303:1 which is less than 4.5:1. 

### Actual Result: 
The MongoDB text, displayed in grey color on a white background, has a contrast ratio below 4.5:1. This makes the text difficult to perceive for users with low vision or those in environments with poor lighting. 
NOTE: Issue is Repro for GitHub Copilot for Azure, Remote development, Jupyter, Gitlens, C#Dev Kit.
### Expected Result: 
The text color should have a contrast ratio of at least 4.5:1 against the white background for normal-sized text and 3:1 for large text. This ensures readability for users with low vision or contrast sensitivity. 
### User Impact: 
Users with low vision may encounter challenges if the color contrast for MongoDB text is insufficient. Specifically, the grey text on a white background currently has a contrast ratio of 2.303:1, which does not meet the recommended minimum of 4.5:1. This low contrast can make it difficult for individuals with visual impairments to read the content clearly. Improving the color contrast will help ensure better accessibility and a more inclusive user experience for everyone. 
### Attachment:

<img width="960" height="540" alt="Image" src="https://github.com/user-attachments/assets/b642f344-52ab-496d-93f4-81056157397c" />

Similar issue 

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/7bcb2e40-ba05-4677-afef-7e1fb67e13b6" />

**Comment by hawkticehurst** [comment]
Hi @AnKushSingh05! The contrast failure on the MongoDB (and other VS Code extension cards) is the result of an intentional artistic decision to add a subtle fade to the bottom of the container that the card are inside of.

The fade is supposed to visually imply that the list of extension cards keeps going as a method of indicating that there's a vast selection of 80,000+ VS Code extensions to pick from. In this case, the important thing to indicate is this large selection of extensions rather than the individual information any of the cards contains being fully readable.

<img width="1053" height="544" alt="Image" src="https://github.com/user-attachments/assets/f459e2c8-5b76-4ff2-9f49-991bc84e671d" />

**Comment by hawkticehurst** [comment]
Fixed! We removed the extension grid fade. Should be deployed to the website within the next few days

**Comment by hawkticehurst** [comment]
Reopened for testing

**Comment by kapilvaishna** [comment]
Issue is not repro so closing the bug.
#Closed; #A11yFixed; #Regressed:04-16-2026;

<img width="1279" height="623" alt="Image" src="https://github.com/user-attachments/assets/c5da9051-c687-46a2-950e-4ac8b49d86f1" />
